File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/saf/SafJournalpostQueryService.java
package no.nav.tilbakemeldingsmottak.consumer.saf;

import no.nav.tilbakemeldingsmottak.consumer.saf.journalpost.Journalpost;

public interface SafJournalpostQueryService {

    Journalpost hentJournalpost(String journalpostid, String authorizationHeader);

}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/saf/SafJournalpostQueryService.kt
package no.nav.tilbakemeldingsmottak.consumer.saf

import no.nav.tilbakemeldingsmottak.saf.generated.hentjournalpost.Journalpost

interface SafJournalpostQueryService {
    fun hentJournalpost(journalpostid: String): Journalpost
}
